test/fixtures/prViewPreviewWithMetadataByNumber.json

```diff
@@ -10,6 +10,39 @@
         "author": {
           "login": "nobody"
         },
+        "reviewRequests": {
+            "nodes": [
+                {
+                    "requestedReviewer": {
+                        "__typename": "user",
```

Comment:
I think it might be valuable to also display teams among requested reviewers. But I also think that this is something that could just as well be a follow-up.

Reply:
Let me create another follow-up PR for `team`! Since the key is different from `user`, I guess I need to tweak the sorting logic along with a query update.

Reply:
@mislav 
I've added the team reviewer support https://github.com/cli/cli/pull/762/commits/d70358ea3445cec9d2f341c0448abab966610d8b to this PR.

The following test results are made by below permissions (`read:org` works 👍 ).
```diff
-       q.Set("scope", "repo")
+       q.Set("scope", "repo read:org")
```

It seems that the requested team reviewer will be gone once one of the team members takes an action. The requested team will not be shown in the review results too because a result (`Comment`/`Approved`/`Request changes`) is always recorded with user account as far as I can see.

![Screen Shot 2020-04-22 at 22 05 45](https://user-images.githubusercontent.com/5877477/79985362-7562ae00-84e5-11ea-92a0-dc62bff2974c.png)


I also realized that `ghost` account does not have `Login` value. I added a workaround to show `ghost`'s review results https://github.com/cli/cli/pull/762/commits/67907c8b004fc5331900119b9882ca481b6cc68d.

![Screen Shot 2020-04-22 at 22 17 05](https://user-images.githubusercontent.com/5877477/79986457-07b78180-84e7-11ea-98ac-d19a00c34b6f.png)

I think this edge case is happening for other commands which display a user account. 🤔